presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -3926,13 +3926,16 @@ public void testShowFunctions()
         });
 
         assertTrue(functions.containsKey("avg"), "Expected function names " + functions + " to contain 'avg'");
-        assertEquals(functions.get("avg").asList().size(), 2);
-        assertEquals(functions.get("avg").asList().get(0).getField(1), "double");
-        assertEquals(functions.get("avg").asList().get(0).getField(2), "bigint");
+        assertEquals(functions.get("avg").asList().size(), 3);
+        assertEquals(functions.get("avg").asList().get(0).getField(1), "T");
```

Comment:
This is testing the result of "SHOW FUNCTIONS" SQL ... we would have to change the output of that.